src/signTx.c

```diff
@@ -40,7 +40,7 @@ static inline void initTxAuxDataCtx()
 {
 	explicit_bzero(&ctx->txPartCtx, SIZEOF(ctx->txPartCtx));
 	{
-		ins_sign_tx_aux_data_context_t* auxDataCtx = AUX_DATA_CTX();
+		ins_sign_tx_aux_data_context_t* auxDataCtx = AUX_DATA_CTX;
```

Comment:
Seems we don't need this variable anymore?

Reply:
It's still a function call, hidden in the macro.

Reply:
Hmm, I would not be concerned about speed here, more about readability. I am not sure what the compiler does with in the end (e.g. inlining the function call everywhere vs. storing it in a variable and calling only once) --- perhaps try it with the variable, then without, and tell me what the resulting app size is (does the -O3 gcc option affect it? --- careful, the option is already included in Makefile.defines, so it has to be removed not only in our Makefile, but also there) --- if it is the same, I'd go without the variable.